examples/xonotic/Makefile

```
@@ -41,6 +41,10 @@ image_tag = $(REPOSITORY)/xonotic-example:0.8
 build:
 	cd $(root_path) && docker build -f $(project_path)/Dockerfile --tag=$(image_tag) .
```

```suggestion
	docker build -f $(project_path)/Dockerfile --tag=$(image_tag) .
```

We no longer need all of Agones to build it, so we can avoid a big Docker context pull now!

STK example:
https://github.com/googleforgames/agones/blob/15e90007006df4b0ccde8e47a3ab2b66d38de8cf/examples/supertuxkart/Makefile#L40-L41

Other than that, I think this is good to go (I should test making it, but I'll do that one this is complete).

Not for this PR - but I noticed that `make build` doesn't build the windows image -- we should probably track that work somewhere.

---

Thanks for the suggestion 😃. Please see if it looks good now.